Update weather module tests

```
======================================================================
FAIL: test_locations (phenny.modules.test.test_weather.TestWeather)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/apertium/phenny/web.py", line 62, in wrapper
    return fn(*args, **kw)
  File "/home/travis/build/apertium/phenny/modules/test/test_weather.py", line 47, in test_locations
    check_location(result, expected)
  File "/home/travis/build/apertium/phenny/modules/test/test_weather.py", line 20, in check_location
    self.assertAlmostEqual(result[0], expected[0], places=1)
AssertionError: 41.7533819 != 41.7 within 1 places (0.053381899999997984 difference)
```

--- a/modules/test/test_weather.py
+++ b/modules/test/test_weather.py
@@ -17,8 +17,8 @@ class TestWeather(unittest.TestCase):
     @catch_timeout
     def test_locations(self):
         def check_location(result, expected):
-            self.assertAlmostEqual(result[0], expected[0], places=1)
-            self.assertAlmostEqual(result[1], expected[1], places=1)
+            self.assertAlmostEqual(result[0], expected[0], delta=0.07)
+            self.assertAlmostEqual(result[1], expected[1], delta=0.07)
 
         locations = [
             ('92121', (32.9, -117.2)),

